ai/core/src/unitycatalog/ai/core/databricks.py

```diff
@@ -297,9 +297,15 @@ def create_function(
     @retry_on_session_expiration
     @override
     def create_python_function(
-        self, *, func: Callable[..., Any], catalog: str, schema: str, replace: bool = False
+        self,
+        *,
+        func: Callable[..., Any],
+        catalog: str,
+        schema: str,
+        replace: bool = False,
+        dependencies: Optional[list[str]] = None,
```

Comment:
Does dependencies accept `numpy==x.y.z` ? We might want to include its format in the docstring

Reply:
Yep, standard PYPI formats are all supported :) https://e2-dogfood.staging.cloud.databricks.com/editor/notebooks/604107452029490?o=6051921418418893#command/5622463425235518